### ISSUE TITLE
possible wrong tab escape

### ISSUE THREAD

**Issue by 43501474** [ilikenwf/apt-fast#65]
near line #215 in get_mirrors(), there is such a line:

```
echo -e "${list// /${filepath}\t}$filepath\n"
```

In my test, it produces such wrong output when multiple mirrors are configed:

```
<mirror1_url>t<mirror2_url>t...
```

change it to the following form solve the issue:

```
echo -e "${list// /${filepath}\\t}$filepath\n"
```

**Comment by zhangpy** [comment]
+1
confirmed on my ubuntu14.04 on master branch

**Comment by Arthur2e5** [comment]
Or something like `$'\t'`. A literal tab character will do too -- personally I don't like the idea of spawning `-e` everywhere.

**Comment by chrstphrchvz** [comment]
This was fixed in PR #103 (866c171)
